test/unit/core/data-structures/test-priority-queue.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import PriorityQueue from '../../../../src/core/data-structures/priority-queue';
+import PriorityQueue from '#core/data-structures/priority-queue';
```

Comment:
Imports like this really show how much cleaner imports can be, and how much easier it'll be to reorganize files when you don't have to update `../../../` to `../../../../../src/core` etc